packages/orbit-components/src/Seat/index.tsx

```diff
@@ -22,12 +22,13 @@ const Seat = ({
   id,
   price,
   label,
-  title = "Seat",
-  description = "Presents options for seating",
+  title,
+  description,
+  "aria-labelledby": ariaLabelledBy = "",
 }: Props) => {
```

Comment:
Since the Seat is selectable and you are writing about selected state in the docs, I was looking a bit at the `aria-selected` and my question might be stupid but I have to ask :D So the idea is that element wrapping <Seat /> would have the `aria-selected` prop and it would be the responsibility of the teams?

Reply:
I thought of that. I even considered making the Seat a Radio or Checkbox.
But I feel that is a bit too much complexity here. Also, the `aria-selected` is only applicable to `gridcell`, `option`, `row` and `tab` roles.

So I'd really not explore that path here. I feel that as long as the screen reader announces clearly (and we have three props for that!) the component is pretty accessible.